bing_mining_2.py

```diff
@@ -0,0 +1,94 @@
+'''
+    Sarah Barden and Alisha Pegan
+    March 7th, 2017
+    SoftDesign
+    Data Visualization
+    Code to extract news articles from Bing regarding where Hillary Clinton
+    and Michelle Obama have traveled
+'''
+import json
+import requests
+from bing_knock import BING_KEY
+import indicoio
+from indico_knock import Indico_KEY
+indicoio.config.api_key = Indico_KEY
+# dir({})
+
+
+def bing_search(query, name):
+    # base API url
+    url = 'https://api.cognitive.microsoft.com/bing/v5.0/news/search'
+    # url += '?$format=json&$top=1&Query=%27{}%27'.format(quote_plus(query))
+    # count max at 100
+    # url += '?q={}&count=10'.format(quote_plus(query))
+    url += '?q={}&count=100'.format((query))
+    headers = {'Ocp-Apim-Subscription-Key': BING_KEY}
+    r = requests.get(url, headers=headers)
+    resp = json.loads(r.text)
+    # print(len(resp['value']))
+    # print(resp)
+    news = []
+    # after studying the json, learned what the data structure is
```

Comment:
These comments are more like notes, you should target them towards someone who is reading your code for the first time.